docs/how-to/sandbox.md

```diff
@@ -2,19 +2,24 @@
 
 # How to use Jina Sandbox Executors
 
-Jina Sandbox allows you to try out different Jina Executors online and see how they work together without downloading all the Executors to your local machine.
+It is often the case, that a Jina Executor is just a docker image that contains some logic to process documents. Therefore, you can run it locally if you have docker installed. But the docker image could be up to several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.
 
-## Prerequisites
+Jina Sandbox provides a way to make the downloading and running happen in a cloud environment. It will get a pair of host and port, which you can connect with. Jina will automatically take care of this connection.
```

Comment:
```suggestion
Jina Sandbox provides a way to make the downloading and running happen in a cloud environment. It will give back a pair of host and port, which you can connect with. Jina will automatically take care of this connection.
```